builders/Dockerfile.cross

```diff
@@ -1,5 +1,5 @@
 # We don't use this Rust version but need an image with a Rust environment
-FROM rust:1.47.0-buster
+FROM rust:1.49.0-buster
```

Comment:
Why 1.49 here and 1.47 below?

I would keep them the same. Now that we don't have nightly, we can just use eg. rust:1.47-buster and then remove `rustup default 1.47`.

Or update it to 1.49 all across the board (but I think 1.47 is better tested)

Reply:
Good question. I wanted 1.47 but not an outdated OS image. But I guess we can simplify it here.

I tried 1.48 and 1.49 but both led to a missing rustfmt component. Maybe we just remove the `rust-toolchain` file and only pin the toolchain in builders and CI. Then we don't depend on mac development components like rls, rustfmt and clippy.

Reply:
rust-toolchain is only really important for nightly, we can definitely remove it if it makes issues.

I wouldn't worry about a 3 month old OS image in the CI scripts, unless there is some major issue there

Reply:
Yeah, you're right. Done in addf99155c45fb3b38ffbaaf4d3effe89f0a777d